test-case/check-alsabat.sh

```diff
@@ -33,8 +33,11 @@ OPT_HAS_ARG['r']=1             OPT_VAL['r']=48000
 OPT_NAME['c']='pcm_c'      	OPT_DESC['c']='pcm for capture. Example: hw:1,0'
 OPT_HAS_ARG['c']=1             OPT_VAL['c']=''
 
-OPT_NAME['f']='frequency'       OPT_DESC['f']='target frequency'
-OPT_HAS_ARG['f']=1             OPT_VAL['f']=821
+OPT_NAME['f']='format'       OPT_DESC['f']='target format'
+OPT_HAS_ARG['f']=1             OPT_VAL['f']="S16_LE"
+
+OPT_NAME['F']='frequency'       OPT_DESC['F']='target frequency'
+OPT_HAS_ARG['F']=1             OPT_VAL['F']=821
```

Comment:
Will the change of `-f` break the backward compatibility? For example, daily MTL test runs for different frequencies with `-f`.

Reply:
Yes, so we need to update the alsabat test config in web accordingly when merging this PR.